nautobot/data_validation/form_mixin.py

```diff
@@ -0,0 +1,25 @@
+from django.utils.html import strip_tags
+
+from nautobot.data_validation.models import RequiredValidationRule
+
+
+class DataValidationModelFormMixin:
```

Comment:
Probably should import this in `nautobot.apps.forms` module so that apps are blessed to use it.